### PR TITLE
[cluster-test] comment twin test out to unblock CI during investigation

### PR DESCRIPTION
**Pull request by snakech**
## Motivation

comment twin test out to unblock CI during investigation

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/website, and link to your PR here.)

### CODE REVIEW

File: testsuite/cluster-test/src/suite.rs [hunk 1]
```diff
@@ -63,10 +64,10 @@ impl ExperimentSuite {
             PerformanceBenchmarkParams::new_nodes_down(10).build(cluster),
         ));
         experiments.push(Box::new(
-            PerformanceBenchmarkParams::new_fixed_tps(0, 10).build(cluster),
+            PerformanceBenchmarkThreeRegionSimulationParams {}.build(cluster),
```
Comment (andll) [inline testsuite/cluster-test/src/suite.rs:67]: Is this change needed?

Comment (snakech) [reply testsuite/cluster-test/src/suite.rs:67]: just wanna make pre-release and perf suites have same order, I forgot to change perf-suite last time
